### PR TITLE
392 swa

### PR DESCRIPTION
**Pull request by armin-vcg**
Script works for reference, but FYI, a tag mistake has 2.0.1-SNAPSHOT in the POM.  The script works from the tag, so generates directories / files with the snapshot name.
README is done, unless changes requested.

### CODE REVIEW

File: local-dev/swa/README.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+This script creates a folder with:
+
+	* A copy of the `VA Secure Code Review Validation Request Form.pdf` to be filled out
+	* A zip of the code taken from a git tag (a release version)
+	* A fresh Fortify FPR derived from the tag
+	* A zip of the local Fortify rulepack that used to scan the code
+
+## When and how do I run the script?
+
```
Comment (abhijitvk) [inline local-dev/swa/README.md]: @armin-vcg : I think we must add pre-requisites to run the script. may be something as below?
1) Installation of Fortify SCA and Apps, the path as configured in defaults/swa-prep.properties file
2) Instructions and default values are tested on MAC

**Comment by abhijitvk** [comment]
Screen Shot 2019-07-12 at 12 26 19 PM

@armin-vcg  - Confused with this line `Review files and folders under /Applications/Fortify/Fortify_SCA_and_Apps_18.20/Core/config`. What are the files under this folder to be reviewed?

Also, just making sure I expect the output folder as below. so essentially the final step would be to create a package as a zip file from under folder `submission-files` 

Screen Shot 2019-07-12 at 12 28 58 PM

**Comment by abhijitvk** [comment]
Screen Shot 2019-07-12 at 1 03 52 PM

Build label shows as `bip-reference-perftest-0.0.1-SNAPSHOT`. Is this correct? Just wondering why it didn't label as `bip-reference-reactor-0.0.1-SNAPSHOT`

File: local-dev/swa/README.md (hunk 1 above)
Comment (armin-vcg) [reply local-dev/swa/README.md]: Added prerequisites section, with instructions. Will show when I push all changes requested.

**Comment by armin-vcg** [comment]
Ready for another round of review.
- Removed dependency on parentpom project (have not deleted the project yet)
- The "fortify" and "fortify-sca" maven profiles coexist, added "fortify-merge" profile
- Added fortify.sh in root folder
- Fixed swa-prep.sh to use new maven profiles, guarantee clean "source zip"
- Updated docs

**Pull request merged**